### PR TITLE
Xinyu HW2 Submission

### PR DESCRIPTION
**Pull request by Wentworthliu123**

### CODE REVIEW

**Comment by dhruvalb** [comment]
Evaluation
----------------------------------------------------------------------------
| Topic					 	|Total	| Score | Comments |
|-----------------------------------------------|-------|-------|----------|
| **Question 1 - Estimate MSE**			|10	|5	| The numeric answer is correct. Discussion - not quite. You can use OLS to predict values but the questions such as model overfit, effect of outliers etc. remain in question. Later you notice the variable effect of how the data is split|
| **Question 2.a - Split the data** 		| 5	| 5	||
| **Question 2.b - Fit training model** 	| 5	| 5	||
| **Question 2.c - Calculate test set MSE** 	|10	|10	||
| **Question 2.d - Compare with Q1** 		|10	|10	||
| **Question 3 - Repeated validation** 		|30	|30	||
| **Question 4 - Compare models** 		|30	|30	||
| **TOTAL**            				|100	|95	||
